.github/dependabot.yml

```diff
@@ -6,6 +6,11 @@ updates:
     interval: daily
     time: "07:00"
     timezone: Europe/London
+  cooldown:
+    default-days: 5
+    semver-major-days: 30
+    semver-minor-days: 7
+    semver-patch-days: 3
```

Comment:
is `include: *` needed or is it all packages by default? Docs arent entirely clear.